docs/data/migration/migration-data-grid-v7/migration-data-grid-v7.md

```diff
@@ -39,25 +39,25 @@ Below are described the steps you need to make to migrate from v7 to v8.
   To revert to the previous behavior, pass `rowSelectionPropagation={{ parents: false, descendants: false }}`.
 - The prop `indeterminateCheckboxAction` has been removed. Clicking on an indeterminate checkbox "selects" the unselected descendants.
 
-### Localization
-
-- If `estimatedRowCount` is used, the text provided to the [Table Pagination](/material-ui/api/table-pagination/) component from the Material UI library is updated and requires additional translations. Check the example at the end of [Index-based pagination section](/x/react-data-grid/pagination/#index-based-pagination).
-
 ### Changes to the public API
 
 - The `apiRef.current.resize()` method was removed.
 - The `<GridOverlays />` component is not exported anymore.
 
-<!-- ### Accessibility
+### Localization
 
-TBD
+- If `estimatedRowCount` is used, the text provided to the [Table Pagination](/material-ui/api/table-pagination/) component from the Material UI library is updated and requires additional translations. Check the example at the end of [Index-based pagination section](/x/react-data-grid/pagination/#index-based-pagination).
 
-### Editing
+### Accessibility
 
-TBD
+- The Grid is more aligned with the WAI-ARIA authoring practices and sets the `role` attribute to `treegrid` if the Data Grid is used with row grouping feature.
 
 ### Other exports
 
+- `ariaV8` experimental flag is removed.
```

Comment:
I think that we should try to sell each breaking change, and even more importantly to give people direction on how they can retain previous behavior.

Here, we can at least do the latter, the former is not that important since the people who already uses this flag, theoretically are already sold on it:

```suggestion
- `ariaV8` experimental flag is removed. It's now the default behavior.
```

done in: dac7c3d0351267298e0d05b733393f3eef81b7e4.